cmd/swarm/secondary.go

```diff
@@ -0,0 +1,72 @@
+package swarm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+const (
+	DefaultProject = "OHSS"
+)
+
+var (
+	products = []string{"Openshift Dedicated", "Openshift Online Pro", "OpenShift Online Starter", "Red Hat OpenShift Service on AWS", "HyperShift Preview"}
+)
```

Comment:
var (
	products = []string{"\"Openshift Dedicated\"", "\"Openshift Online Pro\"", "\"OpenShift Online Starter\"", "\"Red Hat OpenShift Service on AWS\"", "\"HyperShift Preview\""}
)
```suggestion
var (
	products = []string{"Openshift Dedicated", "Openshift Online Pro", "OpenShift Online Starter", "Red Hat OpenShift Service on AWS", "HyperShift Preview"}
)
```